exercises/practice/sudoku/.docs/instructions.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Sudoku is the popular number puzzle of fitting the numbers 1-9 into each column, row,
+and 3x3 block using each exactly once. Write a z3 function that takes a 9x9 list of lists
+containing the sudoku puzzle (row major order), and returns the solution as a 9x9 list of
```

Comment:
I think that "row major order" may be a little confusing to readers. Is there a simpler way to explain this?